docsbox/Dockerfile

```diff
@@ -40,7 +40,9 @@ RUN apt-get -qq update \
     && apt-get -qq clean
 
 # Install exif tool
-ARG EXIF_TOOL_VERSION=12.77
+# https://exiftool.org/history.html
+# The most recent production release is Version 12.76
+ARG EXIF_TOOL_VERSION=12.76
```

Comment:
I remember trying with 12.87 and it works. 
Though is not the most recent version 12.92?

Reply:
There's a comment in https://exiftool.org/history.html

Note: The most recent production release is [Version 12.76](https://exiftool.org/Image-ExifTool-12.76.tar.gz). (Other versions are considered development releases, and are not uploaded to [MetaCPAN](https://metacpan.org/release/Image-ExifTool).)

so thus I changes it to 12.76

Reply:
Ok